app/views/projects/show.html.erb

```diff
@@ -32,6 +32,5 @@
   </div>
 </div>
 
-<%= javascript_include_tag("experiments/dropdown_actions") %>
```

Comment:
@aignatov-bio 
I removed this again. This logic should not be used here at all. 
I updated button for new project (changed it to form)